extensions/positron-python/src/client/positron/manager.ts

```diff
@@ -101,8 +102,41 @@ export class PythonRuntimeManager implements IPythonRuntimeManager {
      * Recommend a Python language runtime based on the workspace.
      */
     async recommendedWorkspaceRuntime(): Promise<positron.LanguageRuntimeMetadata | undefined> {
```

Comment:
It's worth noting that we're deviating from the existing approach of using the upstream logic to recommend a workspace (see the discoverer). In other work, we updated upstream code to also recommend local pyenv venvs based on `.python-version` files for example. There may be other differences as well.

Reply:
We will still use the workspace discoverer if this returns undefined! I've been thinking of this as a preliminary sanity check if we will know the interpreter without having to do any discovery work. It might be worthwhile to look for `.python-version` at this point in time too, though. 

Here is the context for usage in Positron Core:
https://github.com/posit-dev/positron/blob/18562151cd62efa13f680cbc15b249f9cafd44a5/src/positron-dts/positron.d.ts#L678-L690

Reply:
That makes sense 👍